xcp_d/interfaces/prepostcleaning.py

```diff
@@ -134,9 +137,9 @@ def _run_interface(self, runtime):
                 custom_confounds_tsv_dropped = custom_confounds_tsv_undropped.drop(
                     np.arange(volumes_to_drop))
             else:
-                print("No custom confounds were found or had their volumes dropped")
+                LOGGER.warning(("No custom confounds were found or had their volumes dropped."))
```

Comment:
```suggestion
                LOGGER.warning("No custom confounds were found or had their volumes dropped.")
```

Reply:
When I don't check that, it usually crashes for me - adding those lines in helped stop that